### DISCUSSION TITLE
Filament dashboard

### DISCUSSION THREAD

**Discussion by Jijiri89** [filamentphp/filament#7010] [General]
Please can anyone kindly help me to return filament dashboard in the traditional controller.
What I want is that, in laravel filament, assuming I want to create my own controller that I want to return the filament admin dashboard from that controller. Just like the way we return normal view file in the controller or in the web.php.

**Comment by danharrin** [comment]
What are you trying to do? Filament doesnt use controllers, it uses Livewire.

**Comment by Jijiri89** [comment]
Thanks for your response. What I want to do is that, I want instead of
using /admin to login, I want a way in which if you logged as admin the
system should automatically redirect you to filament admin dashboard but if
you log in as user, it should automatically redirect you to user dashboard
(Jetstream dashboard).
To do that in laravel, I usually create two different blade files :one for
user and one for admin. Also I add usertype column to the user table. So I
define a method in the controller that, if the user type is admin (1), it
should redirect to admin dashboard, else it should redirect to use
dashboard.
I want to implement the same using filament and Jetstream auth. But I can't
find filament dashboard file directory. Thanks

**Reply by danharrin** [reply]
That can be be accomplished by binding a new `LoginResponse` to the container, which decides where to redirect the user.

**Comment by Jijiri89** [comment]
Ok, thanks for the clarification. How can I do that please, how do I bind
to the container. Which logic do I need to write to achieve that?

**Reply by danharrin** [reply]
Similar to #6916.

**Comment by Jijiri89** [comment]
Thanks . I will check it out
